src/main.cpp

```diff
@@ -1,6 +1,6 @@
 ////////////////////////////////////////////////////////////////////////////////
 //            Copyright (C) 2004-2011 by The Allacrost Project
-//            Copyright (C) 2012-2016 by Bertram (Valyria Tear)
+//            Copyright (C) 2012-2017 by Bertram (Valyria Tear)
```

Comment:
Just a side note on good git practices: for future commits it's best to avoid modifying unrelated parts of the file, such as here the copyright header. Those three year bumps are of course not enough as it should be done for every single file in the project, so it's best not to do it at all and have it done for all files at once in their own commit.

No need to change how it is now though, the change doesn't hurt ;)